PoshRSJob/Public/Start-RSJob.ps1

```diff
@@ -39,7 +39,10 @@ Function Start-RSJob {
         .PARAMETER PSSnapinsToImport
             A collection of PSSnapins that will be imported into the background runspace job.
 
-        .PARAMETER FunctionsToImport
+        .PARAMETER FunctionFilesToLoad
+            A collection of files containing custom functions that will be imported into the background runspace job.
+
+        .PARAMETER FunctionsToLoad
```

Comment:
hi,

I just cloned the master branch of the repo, if we take a look at the line 178 of :
https://github.com/proxb/PoshRSJob/blob/master/PoshRSJob/Public/Start-RSJob.ps1

The real param name is **FunctionsToLoad** but not **FunctionsToImport**, I just correct it in the help file to be the same as declared in the function param block.

And the new param that I added is : FunctionFilesToLoad